src/main/java/use_cases/LoginCheckUserExists.java

```diff
@@ -0,0 +1,88 @@
+package use_cases;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class LoginCheckUserExists {
+
+    public String username;
+    public String password;
+
+
+    public LoginCheckUserExists(String username, String password)
+    {
+        this.password = password;
+        this.username = username;
+    }
+
+    public List<Boolean> CheckExists(String username, String password)
+    {
+        List<Boolean> BoolList = new ArrayList<>();
+        String path = "C:\\Users\\ken_w\\Desktop\\Uoft 2022-2023\\Fall 2022\\CSC 207 Software design\\Login System\\forms2.csv";
```

Comment:
We should make an agreement of the default input file path otherwise it would be a mess.